### PR TITLE
jwt token generation improvement

### PR DESCRIPTION
**Pull request by registergoofy**
*add some warning comment for those who want to choose their secret
*strictly follow the golang doc for using crypto/rand
*fatal if not enough entropy

### CODE REVIEW

File: pkg/apiserver/middlewares/v1/jwt.go [hunk 1]
```diff
@@ -147,14 +147,16 @@ func NewJWT(dbClient *database.Client) (*JWT, error) {
 		secret []byte
 	)
 
+	//Please be aware that brute force HS256 is possible.
+	//PLEASE choose a STRONG secret
 	secret_string := os.Getenv("CS_LAPI_SECRET")
```
Comment (buixor) [inline pkg/apiserver/middlewares/v1/jwt.go:152]: maybe check here the length of the secret ?

File: pkg/apiserver/middlewares/v1/jwt.go [hunk 2]
```diff
@@ -147,14 +147,16 @@ func NewJWT(dbClient *database.Client) (*JWT, error) {
...
 
 	if secret_string == "" {
-		secret = make([]byte, 8)
-		if n, err := rand.Reader.Read(secret); err != nil {
+		secret = make([]byte, 64)
+		if n, err := rand.Read(secret); err != nil {
```
Comment (buixor) [inline pkg/apiserver/middlewares/v1/jwt.go:156]: is 64 arbitrarily chosen or is there any kind of best practice we can refer to ?

**Review by buixor — CHANGES_REQUESTED** [review]
maybe check / warn on secret length

File: pkg/apiserver/middlewares/v1/jwt.go (hunk 2 above)
Comment (registergoofy) [reply pkg/apiserver/middlewares/v1/jwt.go:156]: It's the length used at auth0.

**Review by buixor — APPROVED** [review]
lgtm

**Pull request merged**